simple

```diff
@@ -220,16 +232,46 @@ docker)
 
 ###############################################################################
 validate)
+    (
+	[ -e $cli_conf_dir ] || (
+            mkdir $cli_conf_dir
+        ) || exit
+
+        [ -e $installed_pip ] || (
+            yum install -y python-pip && pip install --upgrade pip &&
+            touch $installed_pip
+        ) || exit
+
+        [ -e $installed_virtualenv ] || (
+            yum install -y python-virtualenv && touch $installed_virtualenv
+        ) || exit
+
+	[ -e $installed_git ] || (
+	    yum install -y git && touch $installed_git
+	) || exit
+	echo $validator_dir
+        [ -e $installed_validator ] || (
+            mkdir -p $validator_dir &&  cd $validator_dir && pwd &&
```

Comment:
why the pwd?

Reply:
I'll get rid of it. Used it for debugging.